allink_core/core/cms_plugins.py

```diff
@@ -423,9 +423,9 @@ def get_fieldsets(self, request, obj=None):
             }),
             ('Spacings', {
                 'fields': [
-                    'project_css_spacings_top_bottom',
                     'project_css_spacings_top',
                     'project_css_spacings_bottom',
+                    'project_css_spacings_top_bottom',
```

Comment:
in my opinion we would also need to change the spacing order on the ContentPlugin, If we change it here. Can we just leave top_bottom first ?

Reply:
Good point. I also think the order should be the same on both plugins.